package.json

```diff
@@ -17,16 +17,24 @@
     "webrtc"
   ],
   "dependencies": {
+    "@expo/config-plugins": "^2.0.0",
     "base64-js": "^1.1.2",
     "cross-os": "^1.3.0",
     "event-target-shim": "^1.0.5",
     "prop-types": "^15.5.10",
     "uuid": "^3.3.2"
   },
+  "devDependencies": {
+    "expo-module-scripts": "^2.0.0"
+  },
   "peerDependencies": {
     "react-native": ">=0.60.0"
   },
   "scripts": {
+    "build:plugin": "tsc --build plugin",
+    "clean:plugin": "expo-module clean plugin",
+    "lint:plugin": "eslint plugin/src/*",
+    "prepare": "yarn clean:plugin && yarn build:plugin",
```

Comment:
Please don't use `yarn` here, not everyone is using it and there is no reason to force users to do so.

Reply:
This script will auto run when you publish packages. So, just change it to `npm run` to fit your use-case. User not need to run this/depends on what the user use.